acceptance-tests/src/main/java/io/blueocean/ath/sse/SSEClientRule.java

```diff
@@ -93,7 +94,7 @@ public void setLogEvents(boolean logEvents) {
     public void connect() throws UnirestException, InterruptedException {
         SecureRandom rnd = new SecureRandom();
         String clientId = "ath-" + rnd.nextLong();
-        HttpResponse<JsonNode> httpResponse = Unirest.get(baseUrl + "/sse-gateway/connect?clientId=" + clientId).asJson();
+        HttpResponse<JsonNode> httpResponse = Unirest.get(baseUrl + "/sse-gateway/connect?clientId=" + clientId).basicAuth(LoginPage.getUsername(), LoginPage.getPassword()).asJson();
```

Comment:
might need to add some logic when to do this

Reply:
if running unauthed I guess it may ignore those basic headers anyway (maybe...)

Reply:
We can set the logged in user in the state fairly easily (when using @Login)